pyproject.toml

```diff
@@ -5,9 +5,11 @@ description = ""
 authors = ["Josue"]
 
 [tool.poetry.dependencies]
-python = "3.9.12"
+python = "3.9.15"
+fastecdsa = "2.2.3"
```

Comment:
is it still needed?

Reply:
@joshualyguessennd could you check this?

Reply:
some OS versions need to install fastecda manually.